packages/m/microsoft-gsl/package.yml

```diff
@@ -0,0 +1,23 @@
+name       : microsoft-gsl
+version    : 4.0.0
+release    : 1
+source     :
+    - https://github.com/microsoft/GSL/archive/refs/tags/v4.0.0.tar.gz : f0e32cb10654fea91ad56bde89170d78cfbf4363ee0b01d8f097de2ba49f6ce9
+homepage   : https://github.com/microsoft/GSL
+license    : MIT
+component  : programming.library
+summary    : Guidelines Support Library
+description: |
+    The Guidelines Support Library (GSL) contains functions and types that are suggested for use by the C++ Core Guidelines maintained by the Standard C++ Foundation
+libsplit   : no
+builddeps  :
+    - gtest
```

Comment:
Just double checking: This is a builddep, not a checkdep, correct?

Reply:
I think so. There is no `check` step in the recipe.

Reply:
Not sure, but it won't build without it